offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/offers/OfferDbConnector.groovy

```diff
@@ -148,13 +148,17 @@ class OfferDbConnector implements CreateOfferDataSource, FetchOfferDataSource, P
      * @return the id of the stored offer in the database
      */
     private int storeOffer(Offer offer, int projectManagerId, int customerId, int affiliationId){
-        String sqlValues = "VALUE(?,?,?,?,?,?,?,?,?,?,?,?,?)"
+        String sqlValues = "VALUE(?,?,?,?,?,?,?,?,?,?,?,?,?,?)"
         String queryTemplate = OFFER_INSERT_QUERY + " " + sqlValues
         def identifier = offer.identifier
         List<Integer> generatedKeys = []
         Connection connection = connectionProvider.connect()
         log.info("New offer with id: ${offer.identifier}")
         connection.withCloseable {
+
+            String experimentalDesign = null
+            if(offer.experimentalDesign.isPresent()) experimentalDesign = offer.experimentalDesign.get()
+
```

Comment:
NIT: why did the indentation change here?

Reply:
what do you mean? This is code I added.

Reply:
There is an opening closure above

Reply:
It changes in comparison to the code block before it

Reply:
oh did not see this